pkg/acsengine/const.go

```diff
@@ -55,7 +55,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"hyperkube":    "hyperkube-amd64:v1.6.6",
 		"dashboard":    "kubernetes-dashboard-amd64:v1.6.0",
 		"exechealthz":  "exechealthz-amd64:1.2",
-		"addonresizer": "addon-resizer:1.6",
+		"addonresizer": "addon-resizer:1.7",
```

Comment:
Based on discussion earlier today the recommendation was to do one PR for each component. But do we know if each these changes are compatible with older versions of other components? I am guessing the answer is yes but I wanted to check anyways.